### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Developer-piyush**
I have updated readme and working on some more stuffs to  come up with excellent guides!
If there is any reward for this then here is my akash address:
akash1trhmdtnfd3tlltcxengcs9zwvhwt0mc3x7wmv7

### CODE REVIEW

**Review by colinpollen — CHANGES_REQUESTED** [review]
The start of these changes are in HTML and will not work in our markdown formatted file.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,57 @@
----
-description: Akash means "open space" or "sky" in ancient Sanskrit.
----
-
-# Intro to Akash
-
-## What is Akash?
+<!-- PROJECT LOGO -->
+<br />
```
[the remark this answers is not in the document]
Comment (Developer-piyush) [reply README.md:2]: No sir, it works in both html as well as markup! You can have a look at live changes of mine here: https://github.com/Developer-piyush/docs

[the remark this answers is not in the document]
Comment (andy108369) [reply README.md:2]: Can confirm, it does work https://github.com/ovrclk/docs/blob/b0d20b6772a0e702ae5432f260960d59cb8c2db9/README.md

**Comment by andy108369** [comment]
@Developer-piyush there have been multiple [updates](https://github.com/ovrclk/docs/commits/master/README.md) to the README.md file, mind rebasing your PR?

**Pull request closed without merging**